src/crypto/symmkey.h

```diff
@@ -34,10 +34,10 @@ namespace crypto
       memcpy(iv, data.data() + sizeof(tag), sizeof(iv));
     }
 
-    void set_iv_id(uint64_t id)
+    void set_iv_view(uint64_t view)
```

Comment:
Just to check: in PBFT, does the view number start from 0 or 1?

Reply:
in PBFT view starts from 0